SuperSquareBros.cpp

```diff
@@ -558,7 +562,6 @@ float dt;
 uint32_t lastTime = 0;
 
 Surface* sg_icon_image = Surface::load(asset_scorpion_games);
-Surface* background_image = Surface::load(asset_background);
```

Comment:
Ooh, RAM savings. Hmm, deleting the splash icon after using it would save... 2k, maybe not super important then.

Reply:
I guess I could move the splash icon to the spritesheet, since I know I've got space in there